go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/Financial-Times/coco-synthetic-image-publication-monitor
```

Comment:
Couple of notes regarding modules:
- When migrating v2+ projects to modules it is considered good practice to increase the major version number. The latest version of the service is `38.1.1`, so then we should be releasing `v39.0.0`.
- Do not forget that modules require the version tags to start with a `v` (see the previous note).
- Another thing for v2+ projects that use modules is that the module path should be ending with the major version number i.e. `module github.com/Financial-Times/coco-synthetic-image-publication-monitor/v39`. Since the source code of current project is contained in the main package and cannot be imported by another project, it doesn't really matter what we set the module path to, but I would still recommend doing the semantic import versioning at least for the sake of consistency with other projects.